### ISSUE TITLE
Bug:  Only Default Steve Heads drop from all mobs in 1.12.2 v3.8

### ISSUE THREAD

**Issue by SigynLaufeyson** [Serilum/.issue-tracker#444]
## **Information**
Minecraft version: 1.12.2
Forge version: 14.23.5.2854
Environment: Singleplayer

Mod name: Just Mob Heads
Mod version: 3.8

## **Description**
Only default Steve heads drop from mobs, including Wither Skeletons.  This affects all mobs that would otherwise drop their heads.

## **Crash report**
n/a

**Comment by ricksouth** [comment]
![2021-02-17_14 00 29](https://user-images.githubusercontent.com/38502394/108207834-8e99b280-7128-11eb-89f6-e72b184cc166.png)

**Comment by ricksouth** [comment]
It works fine in my environment. Could you provide more information? Mod list, screenshots etc. Thanks!

**Comment by SigynLaufeyson** [comment]
Hi Rick,  Sorry for the delay.  On the plus side, I can tell you it definitely is a mod conflict.  The old version worked fine so I flipped back to it before I posted here.  I reinstalled the new version & noticed an update to the Zoo & Wild Animals (ZAWA) mod (v. 2.1.1) so I installed it at the same time.  Now I get Steve heads with every single modded mob but the vanilla Minecraft mobs all drop their normal heads.  Since I made no other changes to my mod list since posting here, it's save to assume it's ZAWA. 

The screenshot shows the Steve head dropping from Quark's Blizz mob (lol).
![2021-02-24_15 18 57](https://user-images.githubusercontent.com/79115995/109193722-abf10100-7766-11eb-96de-53e1879360ff.png)

Let me know if you's still like my mod list but I think it's safe to assume your mod is fine.
